src/microbe_stage/PatchManager.cs

```diff
@@ -56,7 +58,7 @@ public void ApplyChangedPatchSettingsIfNeeded(Patch currentPatch, bool despawnAl
             // Despawn old entities
             spawnSystem.DespawnAll();
 
-            // And also all timed entities
+            // And also all timed entities and clear SpawnBags
```

Comment:
Isn't this comment change wrong?

Reply:
opps. I think I did have it do that here at one point.